src/daemon/Display.cpp

```diff
@@ -428,7 +427,7 @@ namespace SDDM {
         env.insert(QStringLiteral("XDG_SESSION_CLASS"), QStringLiteral("user"));
         env.insert(QStringLiteral("XDG_SESSION_TYPE"), session.xdgSessionType());
         env.insert(QStringLiteral("XDG_SEAT"), seat()->name());
-        env.insert(QStringLiteral("XDG_VTNR"), QString::number(terminalNewSession));
+        env.insert(QStringLiteral("XDG_VTNR"), QString::number(m_terminalId));
```

Comment:
we don't want to lose track of which VT X11 is running on, they are separate things

Reply:
Addressed.